src/implementations/wrap/wrap.ts

```diff
@@ -193,19 +219,40 @@ export const wrap = async ({
 
   // Phase 1: For each document, read content, digest and write to file
   const schema = await loadSchema(schemaPath);
-  const individualDocumentHashes = await digestDocument(inputPath, intermediateDir, version, unwrap, schema);
+  const individualDocumentHashes = await digestDocument(
```

Comment:
Might want to rename `digestDocument` to something like `wrapIndividualDocumentsToDir`

Reply:
renamed to `wrapIndividualDocuments`